static/src/js/containers/SidebarContainer/Header/Header.scss

```diff
@@ -6,22 +6,22 @@
 .header__site-logo {
   display: flex;
   align-items: flex-end;
-  background-image: url('../../../../../assets/images/app-logo.png');
+  background-image: url('../../../../assets/images/app-logo.png');
```

Comment:
I don't think we need to fix it now, but I am not a huge fan of these relative links